homepage/homepage_updater.go

```diff
@@ -83,7 +83,22 @@ func (hu *HomepageUpdater) GetHomePageUpdateFor(ctx context.Context, userAccessT
 			mappedFeaturedContent = mapper.FeaturedContent(homepageContent, imageObjects)
 		}
 
-		m := mapper.Homepage(lang, mappedMainFigures, releaseCalModelData, &mappedFeaturedContent, homepageContent.ServiceMessage)
+		var mappedAroundONS []model.Feature
+		if len(homepageContent.AroundONS) > 0 {
+			imageObjects := map[string]image.ImageDownload{}
+			for _, fc := range homepageContent.AroundONS {
+				if fc.ImageID != "" {
+					image, err := hu.clients.ImageAPI.GetDownloadVariant(ctx, userAccessToken, "", "", fc.ImageID, ImageVariant)
+					if err != nil {
+						log.Event(ctx, "error getting image download variant", log.ERROR, log.Error(err), log.Data{"around-ons-entry": fc.Title})
```

Comment:
Question:

1. Just checking that logging any error is all we want to do here? Or should we bomb out if err != nil?
2. Assuming logging is all we want to do - I think you need to add a `continue` as image will be nil - assume we should skip on to the next one in the loop?

Reply:
If the image did not exist then it should log that the image fetch has errored then set that image id as nil, so when the renderer tries to get the image further down the line it would replace the nil with a placeholder image. This functionality is the same as the current behaviour of the featured content section and is what we're trying to replicate with the Around ONS section.